packages/runtime/test/runtime.test.ts

```diff
@@ -1042,13 +1045,16 @@ describe('runtime', () => {
     }`
 
     const res2 = await runtime.executeQuery(updatePost, {
-      i: { id: docID, content: { title: 'A different title' } },
+      i: { id: docID, content: { title: 'A different title' }, testList: ['d', 'e'] },
     })
-
-    expect(res2.errors).toBeDefined()
+    
+    expect(res2.errors?.length).toBe(2)
     expect(res2.errors![0].message).toEqual(
       `Variable "$i" got invalid value { title: "A different title" } at "i.content"; Field "title" is not defined by type "PartialPostInput". Did you mean "date"?`,
     )
+    expect(res2.errors![1].message).toContain(
```

Comment:
I don't really like the idea of testing the immutable fields handling based on parsing GraphQL queries, this seems pretty fragile.
I think instead there should be a direct call to the Ceramic server trying to make an update to an immutable field and getting the matching error message, what do you think?

Reply:
Agreed. FYI I'll close this PR, rebasing from main didnt helped, but I created a new branch branch with the changes(missing this last change) and I'll ask for a review on the new one that doesnt seem to have the CI issue